daml-script/runner/BUILD.bazel

```diff
@@ -37,6 +52,7 @@ da_scala_library(
     tags = ["maven_coordinates=com.daml:daml-script-runner:__VERSION__"],
     visibility = ["//visibility:public"],
     deps = [
+        ":admin_package_service_proto_scala",
```

Comment:
Could we not depends on the sources, so no need to publish it.

Reply:
like for https://github.com/digital-asset/daml/blob/858a1a41e760b908ceedf8cf07d42f5e50c437b6/canton/BUILD.bazel#L495